tests/python_tests/test_preemption.py

```diff
@@ -22,7 +22,6 @@ def get_beam_search_seq_len_300() -> GenerationConfig:
     generation_config.num_beam_groups = 3
     generation_config.num_beams = 6
     generation_config.max_new_tokens = 300
-    generation_config.num_return_sequences = 3
```

Comment:
It's redundant because of next line: [generation_config.num_return_sequences = generation_config.num_beams](https://github.com/openvinotoolkit/openvino.genai/pull/601/files#diff-4e21ff1bd3ef863593c6b2857633740b8ce6c92e573de59aa787d4b9e192a263L26)